test/e2e/framework/framework.go

```diff
@@ -303,6 +303,27 @@ func (f *Framework) deleteNamespace(ctx context.Context, ns *corev1.Namespace) {
 	klog.InfoS("Namespace removed.", "Namespace", ns.Name)
 }
 
+func (f *Framework) GetObjectStorageType() ObjectStorageType {
+	return TestContext.ObjectStorageType
+}
+
+func (f *Framework) GetObjectStorageProvider() string {
+	switch TestContext.ObjectStorageType {
+	case ObjectStorageTypeGCS:
+		return "gcs"
+	default:
+		return ""
+	}
+}
```

Comment:
I am unable to review this as it isn't used anywhere so it's impossible to deduce the validity of the conversion logic.

Reply:
> I am unable to review this as it isn't used anywhere so it's impossible to deduce the validity of the conversion logic.

This PR is a prerequisite for https://github.com/scylladb/scylla-operator/pull/1742, so it will be used there. I separated the two for clarity. I see two ways out of this:
1. merge the two PRs into one
2. move the framework part into #1742 

> how is this planned to be tested? the logic seems to be unused

I believe the above also answers this

Reply:
3. seems least disruptive at this point

Reply:
moved getters to https://github.com/scylladb/scylla-operator/pull/1742

Reply:
the file is still here

Reply:
I didn't push the changes then yet, done